Userland/Libraries/LibWeb/CSS/StyleValues/ScrollbarGutterStyleValue.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024, the Ladybird developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibWeb/CSS/StyleValue.h>
+
+namespace Web::CSS {
+
+class ScrollbarGutterStyleValue final : public StyleValueWithDefaultOperators<ScrollbarGutterStyleValue> {
+public:
+    static ValueComparingNonnullRefPtr<ScrollbarGutterStyleValue> create(bool stable, bool both_edges)
+    {
+        return adopt_ref(*new (nothrow) ScrollbarGutterStyleValue(stable, both_edges));
+    }
+    virtual ~ScrollbarGutterStyleValue() override = default;
+
+    bool stable() const { return m_stable; }
+    bool both_edges() const { return m_both_edges; }
+
+    virtual String to_string() const override
+    {
+        if (m_both_edges) {
+            return "stable both-edges"_string;
+        }
+        if (m_stable) {
+            return "stable"_string;
+        }
+        return "auto"_string;
+    }
+
+    bool properties_equal(ScrollbarGutterStyleValue const& other) const { return m_stable == other.m_stable && m_both_edges == other.m_both_edges; }
+
+private:
+    ScrollbarGutterStyleValue(bool stable, bool both_edges)
+        : StyleValueWithDefaultOperators(Type::ScrollbarGutter)
+        , m_stable(stable)
+        , m_both_edges(both_edges)
+    {
+    }
+
+    bool m_stable { false };
+    bool m_both_edges { false };
```

Comment:
You can't get both_edges to true without stable also being true, I imagine there's a better way to store this?